lib/Accordion/Accordion.js

```diff
@@ -89,33 +81,5 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	mixins: options.accessibility ? [AccordionAccessibilitySupport] : null,	
-
-	/**
-	* @private
-	*/
-	classes: 'moon-accordion',
-
-	/**
-	* @private
-	*/
-	components: [
-		{name: 'headerWrapper', kind: Item, classes: 'moon-accordion-header-wrapper', onSpotlightFocus: 'headerFocus', ontap: 'expandContract', components: [
-			// headerContainer required to avoid bad scrollWidth returned in RTL for certain text widths
-			// (webkit bug)
-			{name: 'headerContainer', kind: Control, classes: 'moon-expandable-list-item-header moon-accordion-header', components: [
-				{name: 'header', kind: MarqueeText, accessibilityDisabled: true}
-			]}
-		]},
-		{name: 'drawer', kind: Drawer, resizeContainer:false, classes: 'moon-expandable-list-item-client', components: [
-			{name: 'client', kind: Group, tag: null}
-		]}
-	],
-
-	/**
-	* @private
-	*/
-	bindings: [
-		{from: '.disabled', to: '.$.headerWrapper.disabled'}
-	]
+	classes: 'moon-accordion'
```

Comment:
I wonder if, somewhere down the road, we can deprecate this kind and just make this a special case of `ExpandableListItem` i.e. enabling via a property of that kind.